packs/data/age-of-ashes-bestiary.db/seismic-spears-trap.json

```diff
@@ -99,7 +99,7 @@
                 "attackEffects": {
                     "custom": "",
                     "value": [
-                        "map-does-not-apply",
+                        "no-map",
```

Comment:
I could swear we had a passive on this one that said "MAP does not apply" that was included with this so it would post when making the strike.

Reply:
Some traps do have this passive - see "Shuffling Scythe Blades". Is this the preferred way to handle this? I couldn't find this passive in the compendium.

Reply:
That's the proper way to do it, yeah. So we should add it to any that missed it.